.github/ISSUE_TEMPLATE/design_proposal.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Design Proposal
+about: Propose a new design
+title: ''
+labels: proposal
+assignees: ''
+
+---
+
+**What design is being proposed?**
+Supply a summary of the design, perhaps with a problem statement that the proposal is intending to solve.
+
+**Additional Context**
+Provide further details and context around the proposal. What areas of Operator will it affect? (For example: CRD, controller, etc.)
+
+**Risks/Concerns**
+What are some potential concerns with the proposal? (For example: heavy development cost, extensive refactoring, high complexity.)
+
+**What other approaches were considered?**
+Provide other approaches considered prior to deciding on the submitted proposal.
+
+**Implementation details**
+If applicable, provide potential implementation details assuming the proposal is accepted.
+
+# Checklist
+- [ ] An announcement of this proposal has been sent to the Porter mailing list: https://getporter.org/mailing-list
```

Comment:
These requirements are more for big stuff like PEPs. Let's remove them from this template since most stuff won't need it and we can ask for a PEP if it's really big.